src/mame/drivers/8080bw.cpp

```diff
@@ -738,17 +745,18 @@ INPUT_PORTS_END
 static INPUT_PORTS_START( spaceat2 )
 	PORT_INCLUDE( sicv )
 
+	PORT_MODIFY("IN0")
+	PORT_DIPNAME( 0x04, 0x00, DEF_STR( Difficulty ) )   /* [code: 0x18ca-d1] */
+	PORT_DIPSETTING(    0x00, DEF_STR( Easy ) )
+	PORT_DIPSETTING(    0x04, DEF_STR( Hard ) )
+
```

Comment:
Same for this one – does this bank of DIP switches have locations?

Reply:
Ditto